packages/SettingsProvider/src/com/android/providers/settings/SettingsProviderHooks.java

```diff
@@ -24,7 +24,13 @@ class SettingsProviderHooks {
     static void insertSettingsOnBoot(final SettingsProvider.SettingsRegistry registry, final int userId) {
         if (userId == UserHandle.USER_SYSTEM) {
             SettingsState globalSettings = registry.getSettingsLocked(SettingsProvider.SETTINGS_TYPE_GLOBAL, userId);
+            globalSettings.insertSettingLocked(Settings.Global.ADD_USERS_WHEN_LOCKED, "0" /* (disabled) value */,
```

Comment:
To avoid repeating the same argument values, add a wrapper method that takes SettingsState, key and value as parameters.